plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSplitSource.java

```diff
@@ -91,7 +92,12 @@ public HudiSplitSource(
                 queue,
                 new BoundedExecutor(executor, getSplitGeneratorParallelism(session)),
                 createSplitWeightProvider(session),
-                partitions);
+                partitions,
+                throwable -> {
+                    trinoException.compareAndSet(null, new TrinoException(HUDI_CANNOT_OPEN_SPLIT,
+                            "Failed to generate splits for " + table.getTableName(), throwable));
```

Comment:
Use `table.getSchemaTableName()` so that it includes the schema name

Reply:
Done